### ISSUE TITLE
[DevTools Error] Cannot remove node "527" because no matching node was found in the Store.

### ISSUE THREAD

**Issue by khairmuhammadLU** [facebook/react#21527]
### Website or app

localhost

### Repro steps

The error was thrown at chrome-extension://fmkadmapgofadopljbjfkapdkoienihi/build/main.js:21477:43
    at bridge_Bridge.emit (chrome-extension://fmkadmapgofadopljbjfkapdkoienihi/build/main.js:19585:22)
    at chrome-extension://fmkadmapgofadopljbjfkapdkoienihi/build/main.js:19745:12
    at listener (chrome-extension://fmkadmapgofadopljbjfkapdkoienihi/build/main.js:37733:39)

### How often does this bug happen?

Often

### DevTools package (automated)

react-devtools-extensions

### DevTools version (automated)

4.13.2-e468072e17

### Error message (automated)

Cannot remove node "527" because no matching node was found in the Store.

### Error call stack (automated)

```text
at chrome-extension://fmkadmapgofadopljbjfkapdkoienihi/build/main.js:21477:43
    at bridge_Bridge.emit (chrome-extension://fmkadmapgofadopljbjfkapdkoienihi/build/main.js:19585:22)
    at chrome-extension://fmkadmapgofadopljbjfkapdkoienihi/build/main.js:19745:12
    at listener (chrome-extension://fmkadmapgofadopljbjfkapdkoienihi/build/main.js:37733:39)
```

### GitHub query string (automated)

```text
https://api.github.com/search/issues?q=Cannot remove node  because no matching node was found in the Store. in:title is:issue is:open is:public label:"Component: Developer Tools" repo:facebook/react
```

**Comment by bvaughn** [comment]
Hi @khairmuhammadLU.

I'm sorry you ran into this problem 😦 

Unfortunately, it doesn't look like this issue has enough info for one of us to reproduce it though. This means that **it's going to be very hard for us to fix**.

Please help us by providing a link to a CodeSandbox (https://codesandbox.io/s/new), a repository on GitHub, or a minimal code example that reproduces the problem. (Screenshots or videos can also be helpful if they help provide context on how to repro the bug.)

Here are some tips for providing a minimal example: https://stackoverflow.com/help/mcve

**Comment by bvaughn** [comment]
Also, it is possible that this issue was already fixed by #21523.

If you'd like to verify, download this build:
[ReactDevTools.zip](https://github.com/facebook/react/files/6505445/ReactDevTools.zip)

And follow these steps to install it:
https://user-images.githubusercontent.com/29597/118749019-b6cda600-b82a-11eb-9449-ef56f41eced5.mp4

**Comment by bvaughn** [comment]
I just released version 4.13.3 (to NPM and Chrome/Firefox/Edge). Please give your browser a couple of hours to update to this version, and then try again. I believe it should fix this issue.

If not, please re-open this issue **with steps on how I can reproduce the problem you're seeing** and I would be happy to take a look! :smile:
